src/workshops/HDA_Flowsheet_Optimization/flowsheet_kinetic_OUU.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct  2 13:44:20 2020
+
+@author: andrew
+"""
+
+import argparse
+import pytest
+from pyomo.environ import (Constraint,
+                           Var,
+                           ConcreteModel,
+                           Expression,
+                           Objective,
+                           SolverFactory,
+                           TransformationFactory,
+                           value,
+                           TerminationCondition)
+from pyomo.network import Arc, SequentialDecomposition
+from pyomo.util.infeasible import log_infeasible_constraints, \
+    log_infeasible_bounds
+from idaes.core import FlowsheetBlock
+from idaes.generic_models.unit_models import (PressureChanger,
+                                              Mixer,
+                                              Separator as Splitter,
+                                              Heater,
+                                              CSTR,
+                                              Flash,
+                                              Translator)
+from idaes.generic_models.unit_models.distillation import TrayColumn
+from idaes.generic_models.unit_models.distillation.condenser \
+    import CondenserType, TemperatureSpec
+from idaes.generic_models.unit_models.pressure_changer \
+    import ThermodynamicAssumption
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.initialization import propagate_state
+from idaes.core.util.testing import get_default_solver
+# Import idaes logger to set output levels
+import idaes.logger as idaeslog
+
+from idaes.generic_models.properties.core.generic.generic_property \
+    import GenericParameterBlock
+from idaes.generic_models.properties.activity_coeff_models.\
+    BTX_activity_coeff_VLE import BTXParameterBlock
+from BTHM_ideal import configuration
+# import hda_reaction as reaction_props
+import hda_reaction_kinetic as reaction_props
+from hda_ideal_VLE import HDAParameterBlock
+
+
+def get_init_model(outlvl=idaeslog.WARNING):
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+
+    m.fs.thermo_params = HDAParameterBlock()
+    m.fs.bt_properties = BTXParameterBlock(default={
+                                           "valid_phase":
+                                           ('Liq', 'Vap'),
+                                           "activity_coeff_model":
+                                           "Ideal"})
+
+    m.fs.translator = Translator(default={
```

Comment:
A minor comment for now, but we will eventually want to reorder this to the order we will discuss things as we build the example in the paper. I.e. declare the reaction properties along wit hthe thermo properties, and declare the Transloator and pre-heater unit after the flash.

We will probably also want to rename equipment to have a consistent naming patter, i.e. H102 for the pre-heater, rename the compressor to P101, make the column C101 and standardize the property package names (bthm_params and bt_params were what I used last time).